plugins/base/src/main/kotlin/translators/descriptors/DefaultDescriptorToDocumentableTranslator.kt

```diff
@@ -149,18 +151,19 @@ private class DokkaDescriptorVisitor(
             generics = descriptor.declaredTypeParameters.map { it.toTypeParameter() },
             companion = descriptor.companion(driWithPlatform),
             sourceSets = setOf(sourceSet),
-            extra = PropertyContainer.withAll(
+            extra = PropertyContainer.withAll<DInterface>(
                 descriptor.additionalExtras().toSourceSetDependent().toAdditionalModifiers(),
                 descriptor.getAnnotations().toSourceSetDependent().toAnnotations(),
                 ImplementedInterfaces(info.allImplementedInterfaces.toSourceSetDependent())
-            )
+            ).let { if (isExpect || isActual) it + IsExpectActual else it }
```

Comment:
This seems to be added to almost every Documentable (it not almost), right?
Why would you not want to add this as some basic field to Documentable?

Reply:
It's may be reasonable, will change